### PR TITLE
First attempt at styling

### PR DESCRIPTION
**Pull request by pjayathissa**
* Integrated tachyons css https://tachyons.io/
* Modified content to match WHO style
* Added margins and all that good stuff to make it look good

### New Dependency
Used dependency Tachyons: https://tachyons.io/
Licence: MIT
Summary: css toolkit commonly used in the open-source community as an alternative to bootstrap due to its high inline customizability and readability, thus enabling us to develop the app without writing any custom css. It's sponserd by digital ocean

### CODE REVIEW

File: client/ionic/public/index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Ionic App" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+
+    <!-- css stylesheet -->
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
```
Comment (advayDev1) [inline client/ionic/public/index.html]: all modules should be sourced solely locally

Comment (pjayathissa) [reply client/ionic/public/index.html]: done, brought it in via npm

File: client/ionic/public/index.html [hunk 2]
```diff
@@ -21,6 +21,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Ionic App" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+
```
Comment (advayDev1) [inline client/ionic/public/index.html]: nit: random line

**Comment by pjayathissa** [comment]
Done
> Per contributor rules:
> 
> > For each dependency you add, please acknowledge it in your PR description and describe why you chose it, its license, and a summary of how widely it is used in the open source community (do known apps use it? does it have known backers?)

File: client/ionic/public/index.html (hunk 2 above)
Comment (pjayathissa) [reply client/ionic/public/index.html]: fixed

**Comment by SamMousa** [comment]
Can you add some screenshots?

**Comment by hspinks** [comment]
👍 to screenshots. 

also small note, but in the future, try to use a feature branch in your fork of the repo, as it makes it a little bit tougher to pull down locally with the conflicting `master` branch name. code lgtm

**Comment by pjayathissa** [comment]
Screenshot 2020-03-22 at 09 21 50
Screenshot 2020-03-22 at 09 21 35

**Comment by advayDev1** [comment]
Nice!  But I seem to have lost merge permissions.  Over to @SamMousa who has admin over the repo.

**Pull request merged**